app/views/admins/_form.html.erb

```diff
@@ -50,6 +50,17 @@
               }
           ) %>
 
+      <div class="mt-3 mb-3">
+        <%= form.check_box(
+                :receive_approval_notifications,
+                {
+                    checked: admin.receive_approval_notifications,
+                    id: :receive_approval_notifications,
+                    label: "Receive approval emails"
+                }
+            ) %>
+      </div>
+
```

Comment:
Shall we also allow setting this field from the Invite admin page? It might be a convenient place for the inviter (eg. Praveen) to set the field for other folks in advance.

Reply:
Oh, I mistakenly thought this form would address both, and I didn't need to make any further controller updates. I'll go update the invitations controller to accept this param.